services/src/northbound/src/main/java/org/openkilda/northbound/service/impl/SwitchServiceImpl.java

```diff
@@ -258,4 +262,26 @@ public DeleteMeterResult deleteMeter(String switchId, long meterId) {
         DeleteMeterResponse result = (DeleteMeterResponse) validateInfoMessage(deleteCommand, response, requestId);
         return new DeleteMeterResult(result.isDeleted());
     }
+    
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public PortDto configurePort(String switchId,  int port, PortConfigurationPayload config) {
+        logger.info("status {}, speed {}", config.getStatus(), config.getSpeed());
+        String correlationId = RequestCorrelationId.getId();
+        PortConfigurationRequest request = new PortConfigurationRequest(switchId, 
+                port, config.getStatus(), config.getSpeed());
+        CommandWithReplyToMessage updateStatusCommand = new CommandWithReplyToMessage(
+                request, System.currentTimeMillis(), correlationId, 
+                Destination.CONTROLLER, northboundTopic);
+        messageProducer.send(floodlightTopic, updateStatusCommand);
+
+        Message response = messageConsumer.poll(correlationId);
+        PortConfigurationResponse switchPortResponse = (PortConfigurationResponse) validateInfoMessage(
+                updateStatusCommand, response, correlationId);
+
+        return new PortDto(switchPortResponse.getSwitchId(), switchPortResponse.getPortNo(), 
+                true, null);
```

Comment:
Not sure why do you define here two hardcoded values in response. Message is always empty, result is always success...

Reply:
Updated